DataFormats/L1Scouting/test/TestL1ScoutingFormat.sh

```diff
@@ -10,13 +10,40 @@ file=testL1Scouting.root
 
 cmsRun ${LOCAL_TEST_DIR}/read_L1Scouting_cfg.py --inputFile "$file" || die "Failure using read_L1Scouting_cfg.py $file" $?
 
+# The old files read below were generated as follows.
+#
+# Check out the release in the filename and use it without modification to make
+# files with split level 99 (maximum possible splitting for each product)
+#
+# Then execute:
+#
+#   cmsRun DataFormats/L1Scouting/test/create_L1Scouting_test_file_cfg.py
+#
+# Rename the output file.
+#
+# The versions of the classes are encoded in the filenames in
+# alphabetical order. This order is also the order the classes
+# appear in classes_def.xml.
+#
+# For the split level 0 files, do the exact same thing except
+# add the following to the output module configuration.
+#     "splitLevel = cms.untracked.int32(0)"
+
 # test file for muon, jet, e/gamma and energy sums data formats
-oldFile="testL1Scouting_v3_v3_v3_v3_v3_13_3_0_pre5.root"
+oldFile="testL1Scouting_v3_v3_v3_v3_v3_14_0_0_split_99.root"
+inputfile=$(edmFileInPath DataFormats/L1Scouting/data/$oldFile) || die "Failure edmFileInPath DataFormats/L1Scouting/data/$oldFile" $?
+cmsRun ${LOCAL_TEST_DIR}/read_L1Scouting_cfg.py --inputFile "$inputfile" --bmtfStubVersion 0 || die "Failed to read old file $oldFile" $?
+
+oldFile="testL1Scouting_v3_v3_v3_v3_v3_14_0_0_split_0.root"
```

Comment:
Would the for-loop pattern (used in other tests) be feasible here?

Reply:
Done. The pairs with the same argument are in a loop now. Looks better.